internal/cli/analytics.go

```diff
@@ -39,40 +40,42 @@ func initAnalytics(rootCmd *cobra.Command) error {
 		return err
 	}
 
-	if status == analytics.OptDefault {
-		_, err := fmt.Fprintf(os.Stderr, "Send anonymized usage data to Windmill [y/n]? ")
-		if err != nil {
-			return err
-		}
-
-		buf := bufio.NewReader(os.Stdin)
-		c, _, _ := buf.ReadRune()
-		if c == rune(0) || c == '\n' || c == 'y' || c == 'Y' {
-			err = analytics.SetOpt(analytics.OptIn)
+	if !webview.NewAnalyticsOn() {
```

Comment:
Maybe we move this up in to its own if statement so that we don't have to re-indent the rest of the code?

```suggestion
	if webview.NewAnalyticsOn() {
	  return nil
	}
```

Reply:
Yes! I thought this would leave `analyticsService` unset but I forgot that my other PR got abandoned and that `analyticsService` still gets initialized before we do any of the `OptStatus` checks. Good catch!